src/lib/checkbox/checkbox.ts

```diff
@@ -76,6 +76,8 @@ export class MatCheckboxChange {
   source: MatCheckbox;
   /** The new `checked` value of the checkbox. */
   checked: boolean;
+  /** The original event */
+  event: Event;
```

Comment:
I think that this would look a little weird on consumption `doStuffWithNativeEvent($event.event)`. We could call it `nativeEvent` or `originalEvent`.

Reply:
`originalEvent` sounds nice. I'll update